{
  "artifacts": [],
  "command": "my_main",
  "experiment": {
    "base_dir": "/opt/czl/libs/pymarl/src",
    "dependencies": [
      "munch==2.5.0",
      "numpy==1.21.2",
      "PyYAML==6.0",
      "sacred==0.8.2",
      "torch==1.11.0"
    ],
    "mainfile": "main.py",
    "name": "pymarl",
    "repositories": [
      {
        "commit": "c971afdceb34635d31b778021b0ef90d7af51e86",
        "dirty": true,
        "url": "git@github.com:heheyas/pymarl.git"
      },
      {
        "commit": "c971afdceb34635d31b778021b0ef90d7af51e86",
        "dirty": true,
        "url": "git@github.com:heheyas/pymarl.git"
      },
      {
        "commit": "c971afdceb34635d31b778021b0ef90d7af51e86",
        "dirty": true,
        "url": "git@github.com:heheyas/pymarl.git"
      }
    ],
    "sources": [
      [
        "main.py",
        "_sources/main_2e80f10ea4fdcea08424cfdb34d34977.py"
      ],
      [
        "run.py",
        "_sources/run_ff99f0b44bae52e100e0837c81ebabb8.py"
      ],
      [
        "utils/logging.py",
        "_sources/logging_8a7ca797100650efaf6fac808303964b.py"
      ]
    ]
  },
  "fail_trace": [
    "Traceback (most recent call last):\n",
    "  File \"/opt/czl/anaconda3/envs/harl_modified/lib/python3.7/site-packages/sacred/config/captured_function.py\", line 42, in captured_function\n    result = wrapped(*args, **kwargs)\n",
    "  File \"src/main.py\", line 35, in my_main\n    run(_run, config, _log)\n",
    "  File \"/opt/czl/libs/pymarl/src/run.py\", line 48, in run\n    run_sequential(args=args, logger=logger)\n",
    "  File \"/opt/czl/libs/pymarl/src/run.py\", line 210, in run_sequential\n    logger.print_recent_stats()\n",
    "  File \"/opt/czl/libs/pymarl/src/utils/logging.py\", line 51, in print_recent_stats\n    x[1] = x[1].cpu()\n",
    "TypeError: 'tuple' object does not support item assignment\n"
  ],
  "heartbeat": "2022-04-21T05:55:49.779217",
  "host": {
    "ENV": {},
    "cpu": "Intel(R) Xeon(R) Silver 4214 CPU @ 2.20GHz",
    "gpus": {
      "driver_version": "470.103.01",
      "gpus": [
        {
          "model": "NVIDIA GeForce RTX 3090",
          "persistence_mode": false,
          "total_memory": 24268
        },
        {
          "model": "NVIDIA GeForce RTX 3090",
          "persistence_mode": false,
          "total_memory": 24268
        }
      ]
    },
    "hostname": "dell",
    "os": [
      "Linux",
      "Linux-4.15.0-169-generic-x86_64-with-debian-buster-sid"
    ],
    "python_version": "3.7.11"
  },
  "meta": {
    "command": "my_main",
    "options": {
      "--beat-interval": null,
      "--capture": null,
      "--comment": null,
      "--debug": false,
      "--enforce_clean": false,
      "--file_storage": null,
      "--force": false,
      "--help": false,
      "--loglevel": null,
      "--mongo_db": null,
      "--name": null,
      "--pdb": false,
      "--print-config": false,
      "--priority": null,
      "--queue": false,
      "--s3": null,
      "--sql": null,
      "--tiny_db": null,
      "--unobserved": false,
      "COMMAND": null,
      "UPDATE": [],
      "help": false,
      "with": false
    }
  },
  "resources": [],
  "result": null,
  "start_time": "2022-04-21T05:53:21.960451",
  "status": "FAILED",
  "stop_time": "2022-04-21T05:55:49.787262"
}